src/features.ts

```diff
@@ -68,15 +67,6 @@ export const OPT_IN_FEATURES: Feature[] = [
     description: ['Terraware Accelerator Console access to the document producer tool'],
     disclosure: ['This is WIP'],
   },
-  {
-    name: 'Mocked Species',
-    preferenceName: 'enableMockedSpecies',
-    active: true,
-    enabled: false,
-    allowInternalProduction: false,
-    description: ['Appends mocked data for new fields to species'],
-    disclosure: ['This is WIP'],
-  },
```

Comment:
Just realized I'm still using this flag [here in my PR](https://github.com/terraware/terraware-web/pull/2555/files#diff-1f0f038c2136d32b6ba3a8632ebcbcdd07637e3d7f6e8a2b3fa0b56ddf0fe84fR78) when adding the two mock deliverables. Perhaps that should be a new flag?